<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis Options</string>
  </property>
  <widget class="QGroupBox" name="ScriptBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>680</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>1. Script Directory</string>
   </property>
   <widget class="QTreeView" name="Directories">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>320</width>
      <height>230</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QTableView" name="Scripts">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>20</y>
      <width>340</width>
      <height>230</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="DirQuestionBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>15</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ScriptBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>680</width>
     <height>40</height>
    </rect>
   </property>
   <property name="title">
    <string>3. ETC Options</string>
   </property>
   <widget class="QCheckBox" name="SpawnGatingCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spawn_Gating</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ChildGatingCheck">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Child_Gating</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EtcOptionSaveBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>15</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ScriptBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>680</width>
     <height>230</height>
    </rect>
   </property>
   <property name="title">
    <string>3. Selected Script</string>
   </property>
   <widget class="QPushButton" name="ScriptSaveBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>205</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="SelectedScript">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>20</y>
      <width>670</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hooking</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date Modified</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Path</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="DisSelectBtn">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>204</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScriptQuestionBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>15</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>560</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ESC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
